CHANGELOG.md

```diff
@@ -93,6 +93,11 @@
   Thanks [Michael Kotten](https://github.com/michbeck100) for contributing this change.
   [#10021](https://github.com/Kong/kong/pull/10021)
 
+#### Balancer
+
+- Add a new load-balancing `algorithm` attribute `latency` to the Upstream entity.
```

Comment:
Should explain in one sentence what does this algorithm does to make the changelog entry useful for users.

Reply:
Added, perhaps need some tune.